### PR TITLE
Add security and comms info to agent docs

### PR DESCRIPTION
**Pull request by dedemorton**
Adds information requested in #127 

Note that we do eventually plan to provide a topic that describes the various components in the ingest architecture and how they communicate. However we're waiting until a later release because we are making some big changes to the Fleet server.

@jmikell821 Can you review and see if this meets your team's needs? Please look at the notes flagged from reviewers. 
Sorry this one has taken so long to land in the docs.

### CODE REVIEW

File: docs/en/ingest-management/faq.asciidoc [hunk 1]
```diff
@@ -152,11 +152,45 @@ downloads from the {package-registry} at `epr.elastic.co`. This means that
...
 [[what-is-the-endpoint-package]]
-== What is the {elastic-endpoint-integration} integration in {ingest-manager}?
+== What is the {elastic-endpoint-integration} integration in {fleet}?
 
 The {elastic-endpoint-integration} integration provides protection on your {agent}
 controlled host. The integration monitors your host for security-related events,
 allowing for investigation of security data through the {security-app} in {kib}.
 The {elastic-endpoint-integration} integration is managed by {agent} in the
 same way as other integrations. Try it out! For more information, see the
 {security-guide}/index.html[{elastic-sec} solution documentation].
```
Comment (jmikell821) [inline docs/en/ingest-management/faq.asciidoc:162]:
```suggestion
same way as other integrations. Try it out! For more information, see the
{security-guide}/index.html[{elastic-sec} documentation].
```

File: docs/en/ingest-management/faq.asciidoc [hunk 2]
```diff
@@ -152,11 +152,45 @@ downloads from the {package-registry} at `epr.elastic.co`. This means that
...
+
+[discrete]
+[[how-are-security-to-agent-communications-secured]]
+== How are communications secured between {elastic-sec} and {agent}?
+
+{elastic-sec} connects to the agent over loopback TLS on port 6788.
+{elastic-sec} validates that the agent has root (Linux and macOS) or SYSTEM
+(Windows) permissions.
+
+//REVIEWERS: Unsure here whether we want to say Elastic Endpoint or Elastic
```
Comment (DonNateR) [inline docs/en/ingest-management/faq.asciidoc:172]: Elastic Security right @jmikell821?

File: docs/en/ingest-management/faq.asciidoc [hunk 3]
```diff
@@ -152,11 +152,45 @@ downloads from the {package-registry} at `epr.elastic.co`. This means that
...
+//Security. ^^
+
+//REVIEWERS: In 7.10, does Elastic Security validate the digital signature of
```
Comment (jmikell821) [inline docs/en/ingest-management/faq.asciidoc:175]: Hey, @ferullo - do you know the answer to this question?

File: docs/en/ingest-management/faq.asciidoc [hunk 4]
```diff
@@ -152,11 +152,45 @@ downloads from the {package-registry} at `epr.elastic.co`. This means that
...
+// the agent and vice versa??
+
+[discrete]
+[[how-are-agent-kibana-communications-secured]]
+== How are communications secured between {agent} and {kib}?
+
+{agent} connects to {kib} over TLS and authenticates the certificate
+presented by {kib}. The agent then provides an ApiKey as an authentication
```
Comment (jmikell821) [inline docs/en/ingest-management/faq.asciidoc:183]:
```suggestion
presented by {kib}. The agent then provides an API key as an authentication
```
assuming that's what you meant here?

File: docs/en/ingest-management/faq.asciidoc (hunk 3 above)
Comment (ferullo) [reply docs/en/ingest-management/faq.asciidoc:175]: It does not in 7.10.

Comment (ferullo) [reply docs/en/ingest-management/faq.asciidoc:175]: In 7.10 Agent validates the signature of Endpoint before installing Endpoint. I don't think it validates it after that.

File: docs/en/ingest-management/faq.asciidoc [hunk 5]
```diff
@@ -152,11 +152,45 @@ downloads from the {package-registry} at `epr.elastic.co`. This means that
...
+
+{elastic-sec} connects to the agent over loopback TLS on port 6788.
+{elastic-sec} validates that the agent has root (Linux and macOS) or SYSTEM
+(Windows) permissions.
+
+//REVIEWERS: Unsure here whether we want to say Elastic Endpoint or Elastic
+//Security. ^^
+
+//REVIEWERS: In 7.10, does Elastic Security validate the digital signature of
+// the agent and vice versa??
```
Comment (dedemorton) [inline docs/en/ingest-management/faq.asciidoc:176]:
```suggestion
```

File: docs/en/ingest-management/faq.asciidoc (hunk 2 above)
Comment (dedemorton) [reply docs/en/ingest-management/faq.asciidoc:172]: Janeen confirmed over  chat that this is good.

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**